partiql-eval/src/test/kotlin/org/partiql/eval/internal/util/PartiQLValueComparatorTest.kt

```diff
@@ -0,0 +1,429 @@
+package org.partiql.eval.internal.util
```

Comment:
Tests are taken from [NaturalExprValueComparatorsTest.kt](https://github.com/partiql/partiql-lang-kotlin/blob/partiql-eval-relsort/partiql-eval/src/test/kotlin/org/partiql/lang/eval/NaturalExprValueComparatorsTest.kt) but creates and compares the `PartiQLValue`s explictly rather than passing text strings and running the evaluator.